src/main/java/edu/oakland/production/display02/DisplayCommInterfaceImplementation.java

```diff
@@ -14,6 +15,8 @@
 public class DisplayCommInterfaceImplementation implements DisplayCommInterface {
 
   private DisplayComm2WayInterface comm2Way;
+  private TrackData data;
+  private TrackDataPlotter plotter;
```

Comment:
These don't need to be stored at the class level as they are only used at the method level.
```suggestion
```